src/components/Section.js

```diff
@@ -0,0 +1,287 @@
+        ""
+      )}
+      {courseData.sections.length > 1 ? (
+        <h4 className="sectionNum">{section.code}</h4>
+      ) : (
+        ""
+      )}
+      <Attributes
+        instructors={section.instructors}
+        building={splitLocation(section.location).Building}
+        units={convertUnits(section.minUnits, section.maxUnits)}
+        status={section.status}
+        type={section.type}
+        registrationNumber={section.registrationNumber}
+      />
+      {!courseData.sections.every(
+        (section) => section.notes === courseData.sections[0].notes
+      ) && <SectionDescription>{section.notes}</SectionDescription>}
+
+      <DateSection sortedSectionMeetings={sortedSectionMeetings} />
+      <UtilBar>
+        {section.recitations === undefined ||
+        section.recitations.length === 0 ? (
+          <></>
+        ) : (
+          <ExpandButton
+            onClick={(e) => handleExpandList(e, section.registrationNumber)}
+            onKeyPress={(e) => handleExpandList(e, section.registrationNumber)}
+            role="button"
+            tabIndex={0}
+          >
+            <ExpandMoreOutlined
+              style={{
+                transform: expandedList[section.registrationNumber]
+                  ? "rotate(180deg)"
+                  : "rotate(0deg)",
+                transition: "0.5s",
+              }}
+            ></ExpandMoreOutlined>
+            <span
+              style={{
+                color: grey[700],
+              }}
+            >
+              Show Recitations
+            </span>
+          </ExpandButton>
+        )}
+        <CalendarButton
+          onClick={() =>
+            wishlistCourse({
+              year,
+              semester,
+              course: section,
+            })
+          }
+        >
+          <CalendarTodayTwoTone
+            style={{
+              color: styleStatus(section.status),
+            }}
+          />
+          <span style={{ color: styleStatus(section.status) }}>
+            {changeStatus(section)}
+          </span>
+        </CalendarButton>
+        <CalendarButton
+          onClick={() =>
+            wishlistCourse({
+              year,
+              semester,
+              course: section,
+            })
+          }
+        >
+          <AddBoxTwoTone
+            style={{
+              color: grey[700],
+            }}
+          />
+          <span
+            style={{
+              color: grey[700],
+            }}
+          >
+            Add to Wishlist
+          </span>
+        </CalendarButton>
+      </UtilBar>
+      <Collapse
+        in={
+          expandedList[section.registrationNumber] === undefined
+            ? false
+            : expandedList[section.registrationNumber]
+        }
+        timeout="auto"
+        unmountOnExit
+      >
+        {section.recitations ? (
+          section.recitations.map((recitation, i) => {
+            let sortedRecitationsMeetings = recitation.meetings
```

Comment:
this can be a `const`. also make this greater expression a short circuit with `&&`